EssentialFeed/EssentialFeedTests/Feed/API/FeedItemsMapperTests.swift

```diff
@@ -6,7 +6,7 @@ import XCTest
 import EssentialFeed
 
 class FeedItemsMapperTests: XCTestCase {
-	func test_map_throwsErrorOnNon200HTTPResponse() throws {
+	func test_map_throwsErrorOnNon2xxHTTPResponse() throws {
```

Comment:
Same here as for `FeedImageDataMapper`
```suggestion
	func test_map_throwsErrorOnNon200HTTPResponse() throws {
```

Reply:
done